Update LiveSplit.TheSimpsonsHitAndRun.asl

Hampo opened the pull request:
Adds a check to the `InteriorManager`'s `mState` to pause timer whilst entering and exiting

NOTE: Shouldn't be pulled until the SHAR community decides on this being allowed

zoton2 commented:
Before it is pushed automatically to LiveSplit instances, it also has to be updated in the `Releases` directory too, but I will leave this PR anyway until a decision is made.

Hampo commented:
The poll has now passed. I've updated the copy in the `Release` folder.
Additionally, I've updated the version checking to support all SHAR versions, regardless of NoCD. Tested by ItsTicker.
